client/src/App.tsx

```diff
@@ -20,11 +21,17 @@ function App(): JSX.Element {
           <AuthProvider>
             <SocketProvider>
               <Switch>
-                <Route exact path="/login" component={Login} />
-                <Route exact path="/signup" component={Signup} />
-                <Route exact path="/booking" component={Booking} />
+                <Route exact path="/login">
```

Comment:
I notice that this code is from the previous PR, please adjust the base of the PR to only reflect changes pertaining to this feature :)